setup.py

```diff
@@ -22,7 +22,7 @@
     "install_requires": required,
     "extras_require": {
         "spark": ["pyspark>=2.3.2"],
-        "sqlalchemy": ["sqlalchemy>=1.3.16"],
+        "sqlalchemy": ["sqlalchemy>=1.4.0"],
```

Comment:
If we are requiring sqlalchemy as an `extra_require`,  will need to make it `>=1.4.7` as it includes the bugfix we submitted.

Reply:
@Shinnnyshinshin Do we need to do the same in `requirements-dev-sql.txt`?  Or only here in `setup.py`?  Thanks!

Reply:
@alexsherstinsky I believe it should be both

Reply:
@Shinnnyshinshin I just committed the change -- please review carefully and confirm by approving that it is good.  Thanks so much!

Reply:
@Shinnnyshinshin Please double-check based on our latest discussion.